docs/userguide/aggregated-api-endpoint.md

```diff
@@ -12,9 +12,9 @@ To quickly experience this feature, we experimented with karmada-apiserver certi
 
 ### Step1: Obtain the karmada-apiserver Certificate
 
-For karmada deployed using `hack/local-up-karmada.sh`, you can directly copy it from the `/root/.kube/` directory.
+For Karmada deployed using `hack/local-up-karmada.sh`, you can directly copy it from the `/root/.kube/` directory.
 
-```
+```shell
```

Comment:
By the way, do you know what's the difference between `bash` and `shell`?

Reply:
In github's markdown, the shell and bash seem to be the same in terms of highlighting: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5973

Reply:
Got it. Thanks.